src/CGRA.cpp

```diff
@@ -7,15 +7,15 @@
  * Author : Cheng Tan
  *   Date : Jan 9, 2023
  */
-
+ 
 #include <fstream>
 #include "CGRA.h"
 #include "json.hpp"
 
 using json = nlohmann::json;
 
 CGRA::CGRA(int t_rows, int t_columns, bool t_diagonalVectorization,
-	   bool t_heterogeneity, bool t_parameterizableCGRA,
+	   bool t_heterogeneity, bool t_parameterizableCGRA, int t_pathSupportDim,
```

Comment:
Why is it called `pathSupportDim`? What does it actually mean? number of tiles that support path merging?